tests/test_skinny.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+import os
+
+
+@pytest.mark.skipif(
+    "MLFLOW_SKINNY" not in os.environ, reason="This test is only valid for the skinny client"
+)
+def test_fails_sqlalchemy_import():
```

Comment:
Are there any other beefy libraries that we should assert aren't importable here?

Reply:
flask